astropy/visualization/wcsaxes/core.py

```diff
@@ -617,7 +613,7 @@ def get_tightbbox(self, renderer, *args, **kwargs):
 
         # FIXME: we should determine what to do with the extra arguments here.
         # Note that the expected signature of this method is different in
-        # Matplotlib 3.x compared to 2.x.
+        # Matplotlib 3.x compared to 2.x, but we only support 3.x now.
```

Comment:
If the fix here is simple, let me know how to patch. Otherwise, others can really fix it as future PR.